src/bench/crypto_hash.cpp

```diff
@@ -75,6 +75,26 @@ static void SipHash_32b(benchmark::State& state)
     }
 }
 
+static void SipHash_3b(benchmark::State& state)
+{
+    uint64_t hash = 0;
+    uint64_t k2 = 0;
+    std::vector<uint8_t> in(3, 0xee);
+    while (state.KeepRunning()) {
+        hash = CSipHasher(hash, ++k2).Write(in.data(), in.size()).Finalize();
+    }
+}
+
+
+static void SipHash(benchmark::State& state)
+{
+    uint64_t hash = 0;
+    uint64_t k2 = 0;
+    std::vector<uint8_t> in(BUFFER_SIZE,0);
+    while (state.KeepRunning())
```

Comment:
nit: as new code, should have brackets unless one line
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c

Reply:
Done